pkg/virt-api/webhooks/mutating-webhook/mutators/clone-create-mutator.go

```diff
@@ -37,6 +37,17 @@ import (
 )
 
 type CloneCreateMutator struct {
+	targetSuffix string
+}
+
+func NewCloneCreateMutator() *CloneCreateMutator {
+	return NewCloneMutatorWithTargetSuffix("")
```

Comment:
Looks nice now, thanks.

Any reason to avoid passing here the randomized suffix?
I see no special reason to add a condition later on about it being empty.
```go
func NewCloneCreateMutator() *CloneCreateMutator {
    return NewCloneMutatorWithTargetSuffix(rand.String(randomSuffixLength))
}
```